### PR TITLE
add akroma support

### PR DESCRIPTION
**Pull request by detroitpro**
## Description

Adding support for Akroma ( akroma.io ). Akroma team is working on native oracle solutions.

- homepage: https://akroma.io
- node(s) https://rpc.akroma.io
- explorer: https://akroma.io/explorer
- BIP44/ChainId 200625 https://github.com/satoshilabs/slips/pull/326

## Trezor Support

- PR merged https://github.com/trezor/trezor-mcu/pull/387
- PR merged https://github.com/trezor/trezor-common/pull/177
- Next Trezor firmware will have full Akroma support (including EIP-155 replay protection).

### CODE REVIEW

File: app/scripts/controllers/decryptWalletCtrl.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ var decryptWalletCtrl = function($scope, $sce, walletService) {
         hwRskPath:         "m/44'/137'/0'/0",      // first address : m/44'/137'/0'/0/0
         goPath:            "m/44'/6060'/0'/0",     // first address: m/44'/6060'/0'/0/0
         hwEOSClassicPath:  "m/44'/2018'/0'/0",     // first address: m/44'/2018'/0'/0/0
+        akromaPath:        "m/44'/200625'/0'/0",   // first address: m/44'/200625'/0'/0/0
```
Comment (mkrufky) [inline app/scripts/controllers/decryptWalletCtrl.js]: I'm not a MEW maintainer, but I am following this effort closely and offer some feedback in an effort to speed things along.   In keeping with the existing coding style, I'd consider renaming `akromaPath` to `hwAkromaPath`

File: app/scripts/controllers/decryptWalletCtrl.js [hunk 2]
```diff
@@ -112,6 +113,9 @@ var decryptWalletCtrl = function($scope, $sce, walletService) {
                 case nodes.nodeTypes.EOSC:
                     $scope.HDWallet.dPath = $scope.HDWallet.hwEOSClassicPath;
                     break;
+                case nodes.nodeTypes.Akroma:
+                    $scope.HDWallet.dPath = $scope.HDWallet.akromaPath;
```
Comment (mkrufky) [inline app/scripts/controllers/decryptWalletCtrl.js]: I'd consider renaming `akromaPath` to `hwAkromaPath`

File: app/scripts/directives/walletDecryptDrtv.html [hunk 3]
```diff
@@ -991,6 +991,20 @@ <h4 id="modalTitle" class="modal-title" translate="ADD_Radio_5_Path" style="marg
...
             </div>
 
+            <div class="col-sm-4">
+              <label class="radio small">
+                <input aria-describedby="Path: TREZOR (Akroma) {{HDWallet.akromaPath}}"
+                       ng-change="onHDDPathChange()"
+                       ng-model="HDWallet.dPath"
+                       type="radio"
+                      value="{{HDWallet.akromaPath}}"/>
+                <span ng-bind="HDWallet.akromaPath"></span>
```
Comment (mkrufky) [inline app/scripts/directives/walletDecryptDrtv.html]: I'd consider renaming akromaPath to hwAkromaPath

**Review by mkrufky — COMMENTED** [review]
Good work.  I'm looking forward to seeing this merged.  :-)

**Review by ghost — CHANGES_REQUESTED** [review]
You should edit here also https://github.com/kvhnuke/etherwallet/pull/1976/files#diff-fdb517236c6450fac305b50634838a19R695

**Review by detroitpro — COMMENTED** [review]
Good feedback. Will rename.

**Comment by mkrufky** [comment]
### Trezor Support
Chain ID > 255 support: (merged)
https://github.com/trezor/trezor-mcu/pull/381

### Ledger Support

Full 32bit Chain ID support: (merged)
https://github.com/LedgerHQ/blue-app-eth/commit/8260268b0214810872dabd154b476f5bb859aac0
https://github.com/LedgerHQ/blue-app-eth/commit/74c085ca2d032179fcf205cda1dee38c92877933

Official Ledger Akroma app: (merged)
https://github.com/LedgerHQ/blue-app-eth/pull/19

**Comment by mkrufky** [comment]
One dependency remains open for Ledger support:

### MEW Support
Support 32 bit Chain ID with the latest Ledger app:
https://github.com/kvhnuke/etherwallet/pull/1979

**Comment by kvhnuke** [comment]
Once @gre confirm the changes on #1979 we can merge this in as well

**Review by gamalielhere — APPROVED** [review]

**Comment by gamalielhere** [comment]
@detroitpro can you resolve the conflict?

**Pull request merged**